assets/js/components/HostDetails/HostSummary.test.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { screen, waitFor, act } from '@testing-library/react';
+import { renderWithRouter } from '@lib/test-utils';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+import { faker } from '@faker-js/faker';
+import { clusterFactory } from '@lib/test-utils/factories';
+import HostSummary from './HostSummary';
+
+describe('HostSummary', () => {
+  it('should render the content correctly', async () => {
```

Comment:
I would do the next. In this "standard" test, I would only use 2 IP addresses, and we only test the content

Reply:
Good idea no need for duplication :)